docs/devel/e2e-tests.md

```diff
@@ -332,14 +332,14 @@ $ go run hack/e2e.go -v --down
 ### Debugging clusters
 
 If a cluster fails to initialize, or you'd like to better understand cluster
-state to debug a failed e2e test, you can use the `cluster/log-dump.sh` script
+state to debug a failed e2e test, you can use the `cluster/log-dump/log-dump.sh` script
```

Comment:
https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md

Why do these files exist in the main repo any more?  @chenopis ?